lib/money/ecto_type.ex

```diff
@@ -7,43 +7,46 @@ if Code.ensure_compiled?(Ecto.Type) do
     This type expects you to use a single currency.
     The currency must be defined in your configuration.
```

Comment:
Maybe you should add deprecation note to moduledoc.

Reply:
👍